workflows/microbiome/gene-based-pathogen-identification/README.md

```diff
@@ -0,0 +1,22 @@
+# Gene-based Pathogen Identification
+
+In this workflow, we determine whether the samples are pathogenic or not, by looking for genes known to be linked to pathogenicity or to the pathogenecity character of the organism.
+
+- Virulence Factor (VF): gene products, usually proteins, involved in pathogenicity. By identifying them, we can call a pathogen and its severity level
+
+- Antimicrobial Resistance genes (AMR).
+
+    These type of genes have three fundamental mechanisms of antimicrobial resistance that are enzymatic degradation of antibacterial drugs, alteration of bacterial proteins that are antimicrobial targets, and changes in membrane permeability to antibiotics, which will lead to not altering the target site and spread throughput the pathogenic bacteria decreasing the overall fitness of the host.
+
+In this workflow we:
+
+1. Perform genome assembly to get contigs, i.e. longer sequences, using metaflye (Kolmogorov et al. 2020) then assembly polishing using medaka consensus pipeline and visualizing the assembly graph using Bandage Image (Wick et al. 2015)
+2. Generate reports with AMR genes and VF using ABRicate
+
+## Input Datasets
+- Collection of Pre-Processed Sequenced reads of all samples, ready for further analysis with the other workflows, in a `fastqsanger` or `fastqsanger.gz` format, the output of **Nanopore Preprocessing** workflow.
+
+## Output Datasets
+- FASTA and Tabular files to track genes and visualise our pathogenic identification through out all samples.
+
+This workflow is available to try via our [GTN tutorial](https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html)
```

Comment:
I'm still not sure exactly what you mean by this, the workflow is here and you can install it on any Galaxy instance. Did you mean to say something like:

```suggestion
This workflow is described in details in [this GTN tutorial](https://training.galaxyproject.org/training-material/topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html#preprocessing)
```
?

Reply:
I mean that this exact workflow is part of a training material, and if someone is stuck and don't know how to use it or what to with it for, or even what to try it with test datasets in a real use case example, they can go through the training